softvis3d-frontend/src/classes/Scale.ts

```diff
@@ -1,11 +1,14 @@
 export default class Scale implements SelectOptionValue {
 
+    public description: string;
```

Comment:
Good point - no, we do not change it on runtime. As far as i understood the getter-thing we could no do either a "private _description property with a simple getter method" or a "public readonly description: string;". Both would lead to the same behaviour: scale.description as accessor and and immutable description property. I like the second one better as it does not require to write a getter method.